api/src/main/java/io/opencensus/trace/export/SpanData.java

```diff
@@ -49,6 +50,41 @@
 @AutoValue
 public abstract class SpanData {
 
+  /**
+   * Returns a new immutable {@code SpanData}.
+   *
+   * @deprecated Use the version that contains the {@code Kind}.
```

Comment:
suggestion: you can add a Javadoc link to the new method instead.

Reply:
done.